src/Umbraco.Web/Models/MediaCropDataSet.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Dynamic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Web;
+using Newtonsoft.Json;
+using Umbraco.Core;
+using Umbraco.Core.Dynamics;
+using Umbraco.Core.Models;
+using Umbraco.Core.Serialization;
+using Umbraco.Web.PropertyEditors.ValueConverters;
+
+namespace Umbraco.Web.Models
+{
+    [JsonConverter(typeof(NoTypeConverterJsonConverter<MediaCropDataSet>))]
+    [TypeConverter(typeof(MediaCropDataSetConverter))]
+    [DataContract(Name="mediaCropDataSet")]
+    public class MediaCropDataSet : ImageCropDataSet, IHtmlString, IEquatable<MediaCropDataSet>
+    {
+        [DataMember(Name = "udi")]
+        public Udi Udi { get; set; }
+
+        public IPublishedContent MediaItem { get; set; }
```

Comment:
Just naming this `Content` would be more in line with the rest of Umbraco, especially because it _is_ an `IPublishedContent`.

Reply:
Also, this property is only set when using the `MediaCropperValueConverter`. Adding a fallback is a no-go (would require the use of singletons and additional backing fields), but creating a new inherited class (e.g. `MediaCropDataSetModel`) that's only returned by the value converter might be a little overkill.